get-poetry.py

```diff
@@ -58,6 +58,10 @@
 except ImportError:
     winreg = None
 
+try:
+    u = unicode
```

Comment:
Do we really need this alias?

Reply:
Please, keep it there. Original code was failing on AppVeyor tests, this was the solution to make it working on python 2.7 and 3.x.